monocle/github/pullrequest.py

```diff
@@ -21,6 +21,12 @@
 from dataclasses import dataclass
 from pprint import pprint
 
+from .graphql import Timeout
+
+MAX_TRY = 3
+MAX_BULK_SIZE = 100
```

Comment:
I was not lucky at all with so high bulk size.

Reply:
that's the max and it works from time to time but we adapt the size dynamically.

Reply:
ok, but maybe not so high because we will have a high time-out rate that will slow down the crawling process a lot.